com.github.treagod.spectator.json

```diff
@@ -0,0 +1,64 @@
+{
+  "app-id": "com.github.treagod.spectator",
+  "base": "io.elementary.BaseApp",
+  "base-version": "juno-20.08",
+  "runtime": "org.gnome.Platform",
+  "sdk": "org.gnome.Sdk",
+  "runtime-version": "3.38",
+  "command": "com.github.treagod.spectator",
+  "finish-args": [
+    "--socket=wayland",
```

Comment:
You need network access here.

Reply:
That's embarrassing, no idea how I missed it. =(